.gitignore

```diff
@@ -2,3 +2,9 @@
 out/
 site/
 ContextProject.iml
+/logs/*
+.classpath
+.project
+.settings/org.eclipse.jdt.core.prefs
```

Comment:
This was used in another project from which I copied it, if you can state things more concise, please edit it in a future commit